src/app/layout.js

```diff
@@ -17,16 +16,9 @@ export default function RootLayout({ children }) {
         <a className="usa-skipnav" href="#main-content">
           Skip to main content
         </a>
-        <div className="border-bottom border-white">
-          <Banner />
-        </div>
-        <div className="grid-container desktop:display-flex">
-          <div className="desktop:grid-col-2">
-            <Sidebar />
-          </div>
-          <div className="desktop:grid-col-10 bg-white minh-viewport">
-            <main id="main-content">{children}</main>
-          </div>
+        <Banner />
+        <div className="minh-viewport tablet:margin-x-4 margin-x-1">
```

Comment:
I think we're safe to remove `minh-viewport` as well, that was needed more for the "sticky" stuff that is now removed.

Reply:
Yep, it doesn't seem to have messed anything up by removing it!